addons/project/models/project_update.py

```diff
@@ -113,15 +113,60 @@ def _build_description(self, project):
     @api.model
     def _get_template_values(self, project):
         milestones = self._get_milestone_values(project)
+        profitability_values = self._get_profitability_values(project)
+        show_profitability = (profitability_values
+            and profitability_values.get('account_id')
+            and (profitability_values.get('costs') or profitability_values.get('revenues'))
+        )
         return {
             'user': self.env.user,
             'project': project,
+            'profitability': profitability_values,
+            'show_profitability': show_profitability,
             'show_activities': milestones['show_section'],
             'milestones': milestones,
             'format_lang': lambda value, digits: formatLang(self.env, value, digits=digits),
             'format_monetary': lambda value: format_amount(self.env, value, project.currency_id),
         }
 
+    @api.model
+    def _get_profitability_values(self, project):
```

Comment:
We should create a hook for this method in sale_project to handle non-billable project.
If a user created a project without the billable option enabled, the 'project_profitability' section of the project update is empty. The report should follow the same behavior imo.